java/src/main/java/com/powsybl/python/PyPowsyblApiHeader.java

```diff
@@ -144,6 +144,42 @@ private PyPowsyblApiHeader() {
         GeneratorPointer addressOf(int index);
     }
 
+    @CStruct("battery")
+    interface BatteryPointer extends PointerBase {
```

Comment:
I was about to remove all existing element direct access to only keep data frame way to get/update data.
Do we really need this?

Reply:
no